docs/msw/install.md

```diff
@@ -47,53 +47,72 @@ All makefiles and project are located in build\msw directory.
 Microsoft Visual C++ Compilation       {#msw_build_msvs}
 ----------------------------------------------------------------
 
-* From command line using the provided makefiles:
+### From the IDE
 
-0. Open a "Visual Studio Command Prompt" window shortcut to which
-   must have been installed to the "Start" menu or the "Start" screen
-   by MSVS installation.
+Ready to use project files are provided for VC++ versions 8, 9,
+10, 11, 12, 14, 15 and 16 (also known as MSVS 2005, 2008, 2010, 2012,
+2013, 2015, 2017 and 2019 respectively).
+
+Simply open `wx_vcN.sln` (for N=8, 9, 10, 11, 12, 14, 15 or 16) file,
+select the appropriate configuration (Debug or Release, static or DLL)
+and build the solution. Notice that when building a DLL configuration,
+you may need to perform the build several times because the projects
+are not always built in the correct order, and this may result in link
+errors. Simply do the build again, up to 3 times, to fix this.
+
+
+### From the command line
+
+wxWidgets can also be built from the command line using the provided makefiles.
+
+This needs to be done from the "Visual Studio Command Prompt" window, which can
+be opened using a shortcut installed to the "Start" menu or the "Start" screen
+by MSVS installation.
 
-1. Change directory to \%WXWIN\%\build\msw and type
+In this window, change directory to `%WXWIN%\build\msw` and type
 
         > nmake /f makefile.vc
 
-   to build wxWidgets in the default debug configuration as a static
-   library. You can also do
+to build wxWidgets in the default debug configuration as a static library. You
+can also do
 
         > nmake /f makefile.vc BUILD=release
 
-   to build a release version or
+to build a release version or
 
         > nmake /f makefile.vc BUILD=release SHARED=1 TARGET_CPU=X86
```

Comment:
Are those `TARGET_CPU=` needed? AFAIK, when building x86 from x86 prompt or x64 from x64 prompt, they are not.

I also think crosscompiling, e.g., building 32-bit with `TARGET_CPU=X86`  may not be possible from x64 command prompt and vice versa.

Reply:
I think `X86` is not needed because it's the default, but `TARGET_CPU` (or just `CPU`) must be set to `x64` for 64 bit builds, as otherwise you'd get an error from [wx/msw/chkconf.h](https://github.com/wxWidgets/wxWidgets/blob/0ac8a75e9749c6cd715013c058616dc5d6a52ae3/include/wx/msw/chkconf.h#L19), I think.

Reply:
I never use either build parameter and never get the error. I think this takes care of that
https://github.com/wxWidgets/wxWidgets/blob/0ac8a75e9749c6cd715013c058616dc5d6a52ae3/build/msw/makefile.vc#L1909-L1914

Reply:
Hmm, where does this env var come from? It's not defined in my "MSVS Command Prompt" window.

Reply:
I do have `Platform` environment variable set to either `x64` or `x86` by MSVS 2019 and 2017. I also tried 2008 and 2010 which I have installed only for 32-bit and these two do not set the variable.

So I guess the CPU architecture may be needed to be explicitly set with older MSVS versions and you were correct.